bundle/manifests/elasticsearch-operator.clusterserviceversion.yaml

```diff
@@ -457,7 +457,9 @@ spec:
           - watch
         serviceAccountName: elasticsearch-operator
       deployments:
-      - name: elasticsearch-operator
+      - label:
+          name: elasticsearch-operator
```

Comment:
This is unrelated to the bug from operator-sdk 18.1 right?

Reply:
So I had to update operator-sdk to 1.19.1 in order to remake the manifests, and this was generated after `make bundle` and `make generate`
this is probably in the wrong PR

Reply:
Hmm, okay. Maybe let's remove this part for now and then we might be all good?